{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Connected to MySQL Server version  8.0.26-0ubuntu0.20.04.2\n",
      "You're connected to database:  ('anemp',)\n",
      "MySQL connection is closed\n"
     ]
    }
   ],
   "source": [
    "import mysql.connector\n",
    "from mysql.connector import Error\n",
    "\n",
    "try:\n",
    "    connection = mysql.connector.connect(host='localhost',\n",
    "                                         database='anemp',\n",
    "                                         user='root',\n",
    "                                         password='Monu@1234')\n",
    "    if connection.is_connected():\n",
    "        db_Info = connection.get_server_info()\n",
    "        print(\"Connected to MySQL Server version \", db_Info)\n",
    "        cursor = connection.cursor()\n",
    "        cursor.execute(\"select database();\")\n",
    "        record = cursor.fetchone()\n",
    "        print(\"You're connected to database: \", record)\n",
    "\n",
    "except Error as e:\n",
    "    print(\"Error while connecting to MySQL\", e)\n",
    "finally:\n",
    "    if connection.is_connected():\n",
    "        cursor.close()\n",
    "        connection.close()\n",
    "        print(\"MySQL connection is closed\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting mysql\n",
      "  Downloading mysql-0.0.3-py3-none-any.whl (1.2 kB)\n",
      "Collecting mysqlclient\n",
      "  Downloading mysqlclient-2.0.3.tar.gz (88 kB)\n",
      "\u001b[K     |████████████████████████████████| 88 kB 945 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-802b7ja3\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/\n",
      "    Complete output (15 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    mysql_config --version\n",
      "    mariadb_config --version\n",
      "    mysql_config --libs\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/setup_posix.py\", line 70, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c4c9c236c57746e68b3037d41a054e0f/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/3c/df/59cd2fa5e48d0804d213bdcb1acb4d08c403b61c7ff7ed4dd4a6a2deb3f7/mysqlclient-2.0.3.tar.gz#sha256=f6ebea7c008f155baeefe16c56cd3ee6239f7a5a9ae42396c2f1860f08a7c432 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.2.tar.gz (88 kB)\n",
      "\u001b[K     |████████████████████████████████| 88 kB 654 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-tvj3h72j\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_cd192d891a61472fbdbdc402e495181c/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/0e/68/f79de0e0fcdb041783124b825771e0ba6a3d0893b7b4c4735f130e42af47/mysqlclient-2.0.2.tar.gz#sha256=8df057b08fc27d8f7106bfa997d0a21e2acef017f905f06d6fb0aa6a20d4d2b2 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.1.tar.gz (87 kB)\n",
      "\u001b[K     |████████████████████████████████| 87 kB 896 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-vxmx6afy\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25b3296548246a086b56f4f499a0523/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e1/e5f2b231c05dc51d9d87fa5066f90d1405345c54b14b0b11a1c859020f21/mysqlclient-2.0.1.tar.gz#sha256=fb2f75aea14722390d2d8ddf384ad99da708c707a96656210a7be8af20a2c5e5 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-2.0.0.tar.gz (87 kB)\n",
      "\u001b[K     |████████████████████████████████| 87 kB 1.0 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-ktde6789\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/setup.py\", line 15, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/setup_posix.py\", line 65, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b2e85a84552648c187024c15b8341e7b/setup_posix.py\", line 31, in mysql_config\n",
      "        raise OSError(\"{} not found\".format(_mysql_config_path))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ca/cf/dd8124bf8b3d2ae465db9c8ce35c51650c31784b095f41faf6f8cad987a0/mysqlclient-2.0.0.tar.gz#sha256=201dfcc2a3c8a0859ea399947f1af8abe74b706beba0ebac149180d9d7b0f6e8 (from https://pypi.org/simple/mysqlclient/) (requires-python:>=3.5). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.6.tar.gz (85 kB)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[K     |████████████████████████████████| 85 kB 716 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-xt6xpwbj\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_020fd32335084280abfaf8136524e17f/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/d0/97/7326248ac8d5049968bf4ec708a5d3d4806e412a42e74160d7f266a3e03a/mysqlclient-1.4.6.tar.gz#sha256=f3fdaa9a38752a3b214a6fe79d7cae3653731a53e577821f9187e67cbecb2e16 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.5.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 660 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-uoq3_u4x\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2f8d35832c414a2283e457cd333ec780/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f8/9b/5db9a03e2088a87c26e3e4d4c7f7e8f4c2dbae610f9521cdfac15755a795/mysqlclient-1.4.5.tar.gz#sha256=e80109b0ae8d952b900b31b623181532e5e89376d707dcbeb63f99e69cefe559 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.4.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 738 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-yrahlcx_\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_4ef7f410a78f4acbbb0e0766d20df091/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/4d/38/c5f8bac9c50f3042c8f05615f84206f77f03db79781db841898fde1bb284/mysqlclient-1.4.4.tar.gz#sha256=9c737cc55a5dc8dd3583a942d5a9b21be58d16f00f5fefca4e575e7d9682e98c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.3.tar.gz (86 kB)\n",
      "\u001b[K     |████████████████████████████████| 86 kB 793 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-7fapwcjg\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/\n",
      "    Complete output (12 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    /bin/sh: 1: mariadb_config: not found\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/setup_posix.py\", line 61, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_b25dff9b38824c0988c72b9795c30cd2/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/13/37/6c941aabeb78464a538b7d43266bfae645993779e3ca3e12fe82b4590f52/mysqlclient-1.4.3.tar.gz#sha256=0d3eeb97534c9f42e9b65bd35c1a35fadc2f6d7ea5d9fdbda5339a569156344b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.2.post1.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 604 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-fe23dhlp\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_2c744356dd38493db38f75296933693f/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f4/f1/3bb6f64ca7a429729413e6556b7ba5976df06019a5245a43d36032f1061e/mysqlclient-1.4.2.post1.tar.gz#sha256=f257d250f2675d0ef99bd318906f3cfc05cef4a2f385ea695ff32a3f04b9f9a7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.4.2.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 635 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-0b1jwowg\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_dd9e081b63234dc68710b19bc0839f57/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/73/79/67ddf24ac31b05d741f0ac87fa612c7a11bab1b39b8270ed7344d149f8af/mysqlclient-1.4.2.tar.gz#sha256=b95edaa41d6cc47deecabcdcbb5ab437ad9ae6d8955f5cf10d1847b37e66ef5e (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.1.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 608 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-6bc26_tc\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_8a28fd4ed8e641a6994aef3938047508/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/de/d7/919042fb7ed78a89e21c4e43d5daed90d5df18ec183bae3de493a7d9e3ed/mysqlclient-1.4.1.tar.gz#sha256=a62220410e26ce2d2ff94dd0138c3ecfb91db634464a9afb4c8e6b50f0a67e00 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.4.0.tar.gz (85 kB)\n",
      "\u001b[K     |████████████████████████████████| 85 kB 530 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-26tfz1a3\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/setup_posix.py\", line 51, in get_config\n",
      "        libs = mysql_config(\"libs\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_342cea9bef564b299c1a6fe4eccb9cab/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (_mysql_config_path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/05/73/fac741eef89b0257fb1807de80e96474f40d52739cdbc32c90c4a793e79d/mysqlclient-1.4.0.tar.gz#sha256=efeaacd04feea7d44fef2a5adf4df335b438458f8b89144307dc5a90025761a5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.14.tar.gz (91 kB)\n",
      "\u001b[K     |████████████████████████████████| 91 kB 1.2 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-xt98rfjx\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/setup.py\", line 16, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/setup_posix.py\", line 53, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_0f58c75c682846bdbda9c3f77e0ef482/setup_posix.py\", line 28, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/f7/a2/1230ebbb4b91f42ad6b646e59eb8855559817ad5505d81c1ca2b5a216040/mysqlclient-1.3.14.tar.gz#sha256=3981ae9ce545901a36a8b7aed76ed02960a429f75dc53b7ad77fb2f9ab7cd56b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.13.tar.gz (90 kB)\n",
      "\u001b[K     |████████████████████████████████| 90 kB 1.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-gibluxxi\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/setup.py\", line 18, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/setup_posix.py\", line 53, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6dcfc56cbc094b259fd66b84fbb0f216/setup_posix.py\", line 28, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/ec/fd/83329b9d3e14f7344d1cb31f128e6dbba70c5975c9e57896815dbb1988ad/mysqlclient-1.3.13.tar.gz#sha256=ff8ee1be84215e6c30a746b728c41eb0701a46ca76e343af445b35ce6250644f (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.12.tar.gz (89 kB)\n",
      "\u001b[K     |████████████████████████████████| 89 kB 1.0 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-9rx8xzrx\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_ee6dbceb72604075b210157771f5f87b/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6f/86/bad31f1c1bb0cc99e88ca2adb7cb5c71f7a6540c1bb001480513de76a931/mysqlclient-1.3.12.tar.gz#sha256=2d9ec33de39f4d9c64ad7322ede0521d85829ce36a76f9dd3d6ab76a9c8648e5 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.11.tar.gz (89 kB)\n",
      "\u001b[K     |████████████████████████████████| 89 kB 1.2 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-f0mlbhfk\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_fdb7ccc51c06421788910c9f1c586df0/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/4d/856d1848e9b7fa8577a31c0188592f472e5cbc94f588c2a40ff930533d73/mysqlclient-1.3.11.tar.gz#sha256=9a79b38843d61f651266ff16b9fdcc8830d21f0fb852e4406a19e9a715bdf3e2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.10.tar.gz (82 kB)\n",
      "\u001b[K     |████████████████████████████████| 82 kB 20 kB/s  eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-kwg86gd9\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_24f6c66d4b714b0a8ff5cccf8d9021f5/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/40/9b/0bc869f290b8f49a99b8d97927f57126a5d1befcf8bac92c60dc855f2523/mysqlclient-1.3.10.tar.gz#sha256=452aaf9d2ba1a4fb85c134c88f9b1857852deed63b3b13255d7a65d2c1297262 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.9.tar.gz (81 kB)\n",
      "\u001b[K     |████████████████████████████████| 81 kB 1.4 MB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-hel889_i\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a68d965532b24061971b61a23f89cd94/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/db/f5/c8e1657985c31dda16e434edf5257c31572fa5faacd7e48b1618390e4b18/mysqlclient-1.3.9.tar.gz#sha256=990ccf1e1f15b9a291b811c993dc1c216566714bc14e6581b38665bd61c28c99 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.8.tar.gz (81 kB)\n",
      "\u001b[K     |████████████████████████████████| 81 kB 1.2 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-rn0h05wn\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c9e93b449e204f55ac705dc22d6acf64/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/31/ce/7ea049b3d5929b3cce3104967ccca218a9f054517f0d15dfdaf4e76da2a0/mysqlclient-1.3.8.tar.gz#sha256=c64bc386c0b771ed6ff76de0f52e0f957f98d9e13c83b0ddb821afdbe3f5d970 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.7.tar.gz (79 kB)\n",
      "\u001b[K     |████████████████████████████████| 79 kB 641 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-6gr9rez_\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/setup_posix.py\", line 44, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_352fac6d292a46dea87d859fba31dbca/setup_posix.py\", line 26, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/74/ff/4e964e20b559e55d7afa60fbccc6a560f2adf289813bd3d7eb4eb8a87093/mysqlclient-1.3.7.tar.gz#sha256=c74a83b4cb2933d0e43370117eeebdfa03077ae72686d2df43d31879267f1f1b (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.6.tar.gz (78 kB)\n",
      "\u001b[K     |████████████████████████████████| 78 kB 997 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5tdxbnqe\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_48cd1135546641e0908dc8d345cb63f6/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/09/29/6648563af4b45798ba667f17ea55166d7b0ed8717937c06977a1fdbe7df6/mysqlclient-1.3.6.tar.gz#sha256=1f63a8ab111530a0ff9acaeb118074f11e7564b7c124f2e7fc56fcac641aecf2 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.5.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 566 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-dpd73hnl\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_6e3f4ecbac9b42dba629a4286676f47a/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/82/ea/336daf6c7baa564d0c434ab748218d1a7dc61dfc7066e98b72afd24fd2b9/mysqlclient-1.3.5.tar.gz#sha256=80ea68f6f3c2cfbf35c91adde2b40de1f2c31132118d2dcee048c72423f678bf (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.4.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 755 kB/s eta 0:00:011\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-kk5qh1b5\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_29002e39f3214f96bfd80e49328387e4/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/17/d9/03b08e6a033401335b720806b87596aee8d7f7aa87539713238905c4a8d1/mysqlclient-1.3.4.tar.gz#sha256=a0263383f8ac33ad12d68dc61fa8ccbee41244ce916287c6cee71bf8aea82c5c (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.3.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 691 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-1zizmfnn\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_f92e499feccd4c86a38b9ef08b25b43c/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/26/53/c575db342bfca0213c9243ed2642bfadcd420b18d2f477dc812543e3d83a/mysqlclient-1.3.3.tar.gz#sha256=0f0887882d71bf2a079d9724d9323053cfbee8ab2dcfc989daf97d7ce5be9af9 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading mysqlclient-1.3.2.tar.gz (77 kB)\n",
      "\u001b[K     |████████████████████████████████| 77 kB 671 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-3h1chn8o\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_91a25ed998da4bcda81f18987ce5a09d/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/2d/df/5440ee86bbb248325cdd6e1fc9cbbba365018a0d2d57f673610e020e6b1d/mysqlclient-1.3.2.tar.gz#sha256=e44025830501b9f70f8c2fe8eeff66f0df2df198802f7295801dac199b8236ef (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.1.tar.gz (76 kB)\n",
      "\u001b[K     |████████████████████████████████| 76 kB 777 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-2q76qjjs\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_c25ee0cbb302426ba73d6c9d34d2f158/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6b/ba/4729d99e85a0a35bb46d55500570de05b4af10431cef174b6da9f58a0e50/mysqlclient-1.3.1.tar.gz#sha256=3549e8a61f10c8cd8eac6581d3f44d0594f535fb7b29e6090db3a0bc547b25ad (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading mysqlclient-1.3.0.tar.gz (76 kB)\n",
      "\u001b[K     |████████████████████████████████| 76 kB 620 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-lp48e7qn\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/\n",
      "    Complete output (10 lines):\n",
      "    /bin/sh: 1: mysql_config: not found\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/setup.py\", line 17, in <module>\n",
      "        metadata, options = get_config()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/setup_posix.py\", line 47, in get_config\n",
      "        libs = mysql_config(\"libs_r\")\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysqlclient_a3031647bdaf466b8452a95352354b2e/setup_posix.py\", line 29, in mysql_config\n",
      "        raise EnvironmentError(\"%s not found\" % (mysql_config.path,))\n",
      "    OSError: mysql_config not found\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/6a/91/bdfe808fb5dc99a5f65833b370818161b77ef6d1e19b488e4c146ab615aa/mysqlclient-1.3.0.tar.gz#sha256=06eb5664e3738b283ea2262ee60ed83192e898f019cc7ff251f4d05a564ab3b7 (from https://pypi.org/simple/mysqlclient/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25hCollecting mysql\n",
      "  Downloading mysql-0.0.2.tar.gz (1.9 kB)\n",
      "  Downloading mysql-0.0.1.tar.gz (1.7 kB)\n",
      "Collecting MySQL-python\n",
      "  Downloading MySQL-python-1.2.5.zip (108 kB)\n",
      "\u001b[K     |████████████████████████████████| 108 kB 2.3 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_b0afa1ef08274440914346e99913d476/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_b0afa1ef08274440914346e99913d476/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-t6yf3hv4\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysql-python_b0afa1ef08274440914346e99913d476/\n",
      "    Complete output (7 lines):\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_b0afa1ef08274440914346e99913d476/setup.py\", line 13, in <module>\n",
      "        from setup_posix import get_config\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_b0afa1ef08274440914346e99913d476/setup_posix.py\", line 2, in <module>\n",
      "        from ConfigParser import SafeConfigParser\n",
      "    ModuleNotFoundError: No module named 'ConfigParser'\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/a5/e9/51b544da85a36a68debe7a7091f068d802fc515a3a202652828c73453cad/MySQL-python-1.2.5.zip#sha256=811040b647e5d5686f84db415efd697e6250008b112b6909ba77ac059e140c74 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[?25h  Downloading MySQL-python-1.2.4.zip (113 kB)\n",
      "\u001b[K     |████████████████████████████████| 113 kB 2.1 MB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-ts11bado\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/\n",
      "    Complete output (31 lines):\n",
      "    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.28.tar.gz\n",
      "    Traceback (most recent call last):\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/distribute_setup.py\", line 143, in use_setuptools\n",
      "        raise ImportError\n",
      "    ImportError\n",
      "    \n",
      "    During handling of the above exception, another exception occurred:\n",
      "    \n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/setup.py\", line 7, in <module>\n",
      "        use_setuptools()\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/distribute_setup.py\", line 145, in use_setuptools\n",
      "        return _do_download(version, download_base, to_dir, download_delay)\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/distribute_setup.py\", line 124, in _do_download\n",
      "        to_dir, download_delay)\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_22533821ea5d454d81041b790a2fb2bd/distribute_setup.py\", line 194, in download_setuptools\n",
      "        src = urlopen(url)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 222, in urlopen\n",
      "        return opener.open(url, data, timeout)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 531, in open\n",
      "        response = meth(req, response)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 641, in http_response\n",
      "        'http', request, response, code, msg, hdrs)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 569, in error\n",
      "        return self._call_chain(*args)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 503, in _call_chain\n",
      "        result = func(*args)\n",
      "      File \"/home/subi/anaconda3/lib/python3.7/urllib/request.py\", line 649, in http_error_default\n",
      "        raise HTTPError(req.full_url, code, msg, hdrs, fp)\n",
      "    urllib.error.HTTPError: HTTP Error 403: SSL is required\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/90/5a/ce7bef80825c2188cf507baf57b37516e18dffdb198a6766a597f703059a/MySQL-python-1.2.4.zip#sha256=e405f9d6be33923d428acaa4db4f4470427f1d15ea0d2d82a933449ace26bbd9 (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[?25h  Downloading MySQL-python-1.2.3.tar.gz (70 kB)\n",
      "\u001b[K     |████████████████████████████████| 70 kB 627 kB/s eta 0:00:01\n",
      "\u001b[31m    ERROR: Command errored out with exit status 1:\n",
      "     command: /home/subi/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_5be4e265444b4359a0e0e80cadad3691/setup.py'\"'\"'; __file__='\"'\"'/tmp/pip-install-zmr6nhq9/mysql-python_5be4e265444b4359a0e0e80cadad3691/setup.py'\"'\"';f=getattr(tokenize, '\"'\"'open'\"'\"', open)(__file__);code=f.read().replace('\"'\"'\\r\\n'\"'\"', '\"'\"'\\n'\"'\"');f.close();exec(compile(code, __file__, '\"'\"'exec'\"'\"'))' egg_info --egg-base /tmp/pip-pip-egg-info-g_ntb7_2\n",
      "         cwd: /tmp/pip-install-zmr6nhq9/mysql-python_5be4e265444b4359a0e0e80cadad3691/\n",
      "    Complete output (8 lines):\n",
      "    Traceback (most recent call last):\n",
      "      File \"<string>\", line 1, in <module>\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_5be4e265444b4359a0e0e80cadad3691/setup.py\", line 11, in <module>\n",
      "        from setup_posix import get_config\n",
      "      File \"/tmp/pip-install-zmr6nhq9/mysql-python_5be4e265444b4359a0e0e80cadad3691/setup_posix.py\", line 101\n",
      "        print \"\"\"You shouldn't be running this directly; it is used by setup.py.\"\"\"\n",
      "                                                                                  ^\n",
      "    SyntaxError: Missing parentheses in call to 'print'. Did you mean print(\"\"\"You shouldn't be running this directly)?\n",
      "    ----------------------------------------\u001b[0m\n",
      "\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/9a/81/924d6799494cf7fb24370335c2f782088d6ac4f79e4137d4df94cea3582c/MySQL-python-1.2.3.tar.gz#sha256=7de66fbbf923634e7c965aeaefa74642ba75ae20ee1cefcefc3009595b7a7e6e (from https://pypi.org/simple/mysql-python/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.\u001b[0m\n",
      "\u001b[31mERROR: Cannot install mysql==0.0.1, mysql==0.0.2 and mysql==0.0.3 because these package versions have conflicting dependencies.\u001b[0m\n",
      "\u001b[?25h\n",
      "The conflict is caused by:\n",
      "    mysql 0.0.3 depends on mysqlclient\n",
      "    mysql 0.0.2 depends on mysqlclient\n",
      "    mysql 0.0.1 depends on MySQL-python\n",
      "\n",
      "To fix this you could try to:\n",
      "1. loosen the range of package versions you've specified\n",
      "2. remove package versions to allow pip attempt to solve the dependency conflict\n",
      "\n",
      "\u001b[31mERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies\u001b[0m\n",
      "\u001b[33mWARNING: You are using pip version 21.0.1; however, version 21.2.4 is available.\n",
      "You should consider upgrading via the '/home/subi/anaconda3/bin/python -m pip install --upgrade pip' command.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "!pip install mysql\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
